src/main/java/at/hannibal2/skyhanni/data/IslandType.kt

```diff
@@ -1,40 +1,74 @@
 package at.hannibal2.skyhanni.data
 
-enum class IslandType(val displayName: String) {
-    // TODO USE SH-REPO (for displayName only)
-    PRIVATE_ISLAND("Private Island"),
-    PRIVATE_ISLAND_GUEST("Private Island Guest"),
-    THE_END("The End"),
-    KUUDRA_ARENA("Kuudra"),
-    CRIMSON_ISLE("Crimson Isle"),
-    DWARVEN_MINES("Dwarven Mines"),
-    DUNGEON_HUB("Dungeon Hub"),
-    CATACOMBS("Catacombs"),
-
-    HUB("Hub"),
-    DARK_AUCTION("Dark Auction"),
-    THE_FARMING_ISLANDS("The Farming Islands"),
-    CRYSTAL_HOLLOWS("Crystal Hollows"),
-    THE_PARK("The Park"),
-    DEEP_CAVERNS("Deep Caverns"),
-    GOLD_MINES("Gold Mine"),
-    GARDEN("Garden"),
-    GARDEN_GUEST("Garden Guest"),
-    SPIDER_DEN("Spider's Den"),
-    WINTER("Jerry's Workshop"),
-    THE_RIFT("The Rift"),
-    MINESHAFT("Mineshaft"),
-
-    NONE(""),
-    ANY(""),
-    UNKNOWN("???"),
```

Comment:
Right now the fallback is the raw enum name, I could add smth that just formats the raw name, [e.g.](https://github.com/SkyblockAPI/SkyblockAPI/blob/0ecf9b828b67225be5c5114a6f25f40b5a604826/src/client/kotlin/tech/thatgravyboat/skyblockapi/utils/extentions/StringExtensions.kt#L136)

Reply:
This wouldnt work for things like spiders den or jerrys island, its probably just better to have the actual fallback in the code instead of generating them